app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,11 @@
+from abc import ABC
+from app.players.player import Player
```

Comment:
Divide your imports that use standard libraries and local files with a space next time.